src/pocketmine/lang/locale

```diff
@@ -1 +0,0 @@
-Subproject commit a7d53d9229337323220c7370916a153accc0ebf3
```

Comment:
It is not my intent to change anything in the lang package.  I hit an issue when trying to  do the first pull request.  The build failed as it looked like I was trying to delete several things:

src/pocketmine/lang/locale
src/raklib
src/spl

The problem was that when I created the clone using the fork, it didn't copy over these other directories since it looks like there is a symbolic link to other projects.  Someone else recommended I use git clone --recursive.  Problem with that is that I need to issue a pull request so I assume (maybe wrongly) that I need my own repo on GitHub.  And with GitHub I don't have command line access to do the git clone --recursive.  I can do that from my mac, but then I can't issue a pull request (at least I don't know how to).  So I did do a clone of the pocketmine branch to my mac, and then copied the files over to the local repo pointing to my github repo, and then committed the files and the build was happy.  But it looks like it thinks I changed some files that I didn't.

What is the overall process for creating a clone of the repo and doing a pull request in such a way that it also has the folders which are sym links to other projects?